shell/common/rasterizer.cc

```diff
@@ -382,7 +383,7 @@ RasterStatus Rasterizer::DrawToSurface(flutter::LayerTree& layer_tree) {
   // There is no way for the compositor to know how long the layer tree
   // construction took. Fortunately, the layer tree does. Grab that time
   // for instrumentation.
-  compositor_context_->ui_time().SetLapTime(layer_tree.build_time());
+  compositor_context_->ui_time().SetLapTime(layer_tree.vsync_time());
```

Comment:
@iskakaushik  the `vsync_time` is needed here. I'm not sure what this `SetLapTime` is doing but by using this we are sending in the same value as the original version.

Reply:
Looking at the comment on the top, seems like we should pass the frame build time and remove vsync_time.

Reply:
OK. And it sounds like there will be bunch of benchmark we need to rebase?

Reply:
Removed `vsync_time`.